<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatProdRecipe</class>
 <widget class="QDialog" name="PatProdRecipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>383</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>PAT-Man: PROD Recipe</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pat_prod_recipe.qrc">
    <normaloff>:/pat_prod_recipe/images/images/gex_application.png</normaloff>:/pat_prod_recipe/images/images/gex_application.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButtonNewRecipe">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>32</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>26</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>26</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>New PROD recipe...</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/pat_prod_admin/images/images/script_new.png</normaloff>:/pat_prod_admin/images/images/script_new.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveRecipe">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>32</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>26</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>26</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Save PROD recipe as...</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/pat_prod_admin/images/images/save.png</normaloff>:/pat_prod_admin/images/images/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>122</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Version:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRecipeName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Recipe name</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>72</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Recipe:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRecipeVersion">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>122</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Recipe version</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>152</y>
     <width>601</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Flows</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetFlows">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>581</width>
       <height>151</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Flow</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Source</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Good HBins</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ENG Recipe</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ENG Version</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>File</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButtonPropertiesFlow">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>32</width>
       <height>26</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>26</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Properties: Edit/Review selected flow</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/pat_prod_admin/images/images/properties.png</normaloff>:/pat_prod_admin/images/images/properties.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNewFlow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>32</width>
       <height>26</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>26</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Add flow</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/pat_prod_admin/images/images/new_task.png</normaloff>:/pat_prod_admin/images/images/new_task.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemoveFlow">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>32</width>
       <height>26</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>26</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Remove selected flow...</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/pat_prod_admin/images/images/file_remove.png</normaloff>:/pat_prod_admin/images/images/file_remove.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUp">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>32</width>
       <height>26</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>26</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Move selected flow Up</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/pat_prod_admin/images/images/arrow_up.png</normaloff>:/pat_prod_admin/images/images/arrow_up.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDown">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>32</width>
       <height>26</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>26</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="toolTip">
      <string>Move selected flow Down</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/pat_prod_admin/images/images/arrow_down.png</normaloff>:/pat_prod_admin/images/images/arrow_down.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Binning</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxPatBins">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>PAT Bins</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ZPAT Bin:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxGPAT_Bin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>62</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>GPAT Bin#</string>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxZPAT_Bin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>ZPAT Bin#</string>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxPPAT_Bin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>PPAT Bin#</string>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>GPAT Bin:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PPAT Bin:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Misc.</string>
    </attribute>
    <widget class="QComboBox" name="comboBoxStdf">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enabled/Disable STDF generation</string>
     </property>
     <item>
      <property name="text">
       <string>Disabled</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Enabled: Wafermap only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Enabled: Full STDF</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>76</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>STDF output:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonLoadRecipe">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>32</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>26</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>26</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Load PROD recipe</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/pat_prod_admin/images/images/file_open.png</normaloff>:/pat_prod_admin/images/images/file_open.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditComment">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>97</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Recipe comments</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>97</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Group:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGroup">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Group owning recipe</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAutoIncrement">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>122</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Auti-Increment version when saving recipe</string>
   </property>
   <property name="text">
    <string>Auto-increment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pat_prod_recipe.qrc"/>
 </resources>
 <connections/>
</ui>
